### PR TITLE
[Feature:UI] Align the content in submit box for easier file manipulation

### LINKED ISSUE #4811: Align Content in Submit Box For Easier File Manipulation
**What problem are you trying to solve with Submitty**
Trash cans in submit box are not in line
Results in difficulty deleting multiple files at a time

Screen Shot 2019-11-22 at 3 39 27 PM

**Describe the way you'd like to solve this problem**
Utilize table to organize filename, filesize and delete button

Screen Shot 2019-11-22 at 4 04 20 PM

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)

### What is the current behavior?
Closes #4811 
The Uploaded files are mis-aligned (skewed) which makes them less readable and hard to remove multiple files by clicking on them.
before

### What is the new behavior?
Files are now aligned within the table and there is no such difficulty removing them.
after

### CODE REVIEW

**Comment by mukul-kmr-jha** [comment]
@bmcutler @MasterOdin

File: site/public/css/submitbox.css [hunk 1]
```diff
@@ -35,6 +35,26 @@
...
+}
+
+.fileUploadTable td:nth-child(even){
+    text-align: right;
+}
+  
+.fileUploadTable tr:hover {
+    background: #d1d1d1;
+}
+
```
Comment (MasterOdin) [inline site/public/css/submitbox.css]: The names of classes (and ids) should be in `kebab-case` (e.g. `file-upload-table`).

File: site/public/css/submitbox.css [hunk 2]
```diff
@@ -35,6 +35,26 @@
     height: 150px;
 }
 
+.fileUploadTable {
+    border: none;
+    border-collapse: collapse;
+    text-align: left;
+    margin: 40px auto;
```
Comment (MasterOdin) [inline site/public/css/submitbox.css]: I think we should make this `5px auto`

File: site/public/css/submitbox.css [hunk 3]
```diff
@@ -35,6 +35,26 @@
...
+}
+
+.fileUploadTable td {
+    border: none;
+    padding: 15px 20px;
```
Comment (MasterOdin) [inline site/public/css/submitbox.css]: I think we should make this `5px 10px`

File: site/public/js/drag-and-drop.js [hunk 4]
```diff
@@ -301,40 +301,47 @@ function removeLabel(filename, part){
...
+    // remove file and label-row in table on click event
+    uploadRowElement.children[1].onclick = function(e){
         e.stopPropagation();
         this.parentNode.parentNode.removeChild(this.parentNode);
         deleteSingleFile(filename, part, previous);
     };
 
     // FOR VPAT if trash can has focus and key is pressed it will delete item
-    tmp.children[0].onkeypress = function(e){
+    uploadRowElement.children[1].onkeypress = function(e){
```
Comment (MasterOdin) [inline site/public/js/drag-and-drop.js]: can you simplify all of these `uploadRowElement.children[1]` to just be `fileTrashElement`?

**Comment by mukul-kmr-jha** [comment]
@MasterOdin requested changes are implemented, suggest if anything else need changes.

File: site/public/css/submitbox.css (hunk 2 above)
Comment (mukulkrjha) [reply site/public/css/submitbox.css]: Done

**Comment by MasterOdin** [comment]
@mukul-kmr-jha sorry for the delay on responding. So this PR is good in and of itself, but it breaks usage of the drag and drop interface in the other places it's used (that have different styling as well). I'd like to use this opportunity to investigate the different ways it's being used and unify around one way to do it, after which I'll merge this PR.

**Comment by mukul-kmr-jha** [comment]
@MasterOdin Let me fix and remove these blocks .Thanks!

**Comment by MasterOdin** [comment]
Alright, turns out getting this all working was a bit simpler than I was thinking and I can make an issue on the differences between drag & drop between the different places and a vision of unification. Thanks for your hard work on this!

**Pull request merged**